.github/workflows/code-quality.yml

```diff
@@ -48,15 +48,20 @@ jobs:
       - name: Check Python formatting
         run: black tests --check .
 
-  # Note: in order to keep an `actions/cache` cache up to date, we must
-  # use the approach detailed in
-  # https://github.com/actions/cache/blob/main/workarounds.md#update-a-cache
-  # i.e. load the most recently created cache that matches a prefix,
-  # then create an entirely new cache with every run.
+  build-openassetio:
+    name: Build OpenAssetIO
+    runs-on: ubuntu-latest
+    container:
+      image: ghcr.io/openassetio/openassetio-build
+    steps:
+      - uses: actions/checkout@v3
+      - name: Build
+        uses: ./.github/build_openassetio
 
   cpp-linters:
     name: C++ linters
     runs-on: ubuntu-20.04
+    needs: build-openassetio
     container:
       image: aswf/ci-vfxall:2022-clang13.1
     strategy:
```

Comment:
[fix]/[plan] `strategy` matrix doesn't make much sense since we're using a container.

Reply:
https://github.com/OpenAssetIO/usdOpenAssetIOResolver/pull/16/commits/55bc8822f4c95b01b973a0905e768468531d30b8